template/build/utils.js

```diff
@@ -31,6 +34,10 @@ exports.cssLoaders = function (options) {
 
   // generate loader string to be used with extract text plugin
   function generateLoaders (loader, loaderOptions) {
+    let importLoaders = 0
+    if (options.usePostCSS) importLoaders++
+    if (options.usePostCSS && loader) importLoaders++
```

Comment:
actually I think we don't need this since preprocessors (SASS, LESS, Stylus) will handle `@import` themselves, so there won't be any left for css-loader to care about.
```javascript
const importLoaders = options.usePostCSS && !loader ? 1 : 0
```
that should be enough.